packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -88,6 +88,13 @@ abstract class RenderSliverBoxChildManager {
   /// affect the visible contents of the [RenderSliverMultiBoxAdaptor].
   void setDidUnderflow(bool value);
 
+  /// Called at the beginning of layout to indicate that layout is about to
+  /// occur.
+  void didStartLayout() {}
```

Comment:
Thanks.  I keep forgetting which to use.  (I think we use the opposite of the Google C++ Style guide, which trips me up.)